src/include/meta_properties/pseudopotential.js

```diff
@@ -3,6 +3,10 @@ import _ from "underscore";
 import { Property } from "../../property";
 
 export class Pseudopotential extends Property {
+    static compatibleExchangeCorrelation = {
+        hse06: ["pbe", "hse06"],
```

Comment:
Can we place this somewhere closer to the functional definitions?

Reply:
Let's have a task to resolve this, pls

Reply:
Added task SOF-6942